packages/sw-background-sync-queue/src/lib/background-sync-idb-helper.js

```diff
@@ -0,0 +1,42 @@
+import IDBHelper from '../../../../lib/idb-helper';
```

Comment:
What's the advantage of breaking this out into a separate module vs. keeping an instance of `IDBHelper` as a member variable inside the `RequestQueue` class instances?

It's odd to keep track of the IDB state here inside the `queueObject` module-level variable, when that seems like it should be tied to each instance directly.

I think you could get rid of most of this code that way, and just call the direct `.get()`/`.put()` methods on `IDBHelper`.